bundle/cert-manager-operator/0.15.0-patch.1/certmanagers.operator.cert-manager.io.crd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Don't think this file should be in this PR

Reply:
@meyskens Is this here for the same reason as the clusterserviceversion.yaml (above) ?